README.md

```diff
@@ -1,6 +1,15 @@
 puppetdb
 =========
 
+Statement of Support - Maintenance Mode
+---------------------------------------
+
+> This module is maintained by Puppet, but we have no plans for future feature development. We
```

Comment:
Sorry but this must be a joke. Why doesn't Perforce want to provide puppet modules for their own software?
And what about the puppetdb specific code in pe-modules rpm/deb package? That's probably still maintained? Why can't Puppet Enterprise eat their own dogfood and use the puppetlabs/puppetdb module instead?

You put us now in a situation where open source users are treated like second class citizens and I fail to understand why.

Reply:
Another thought: There even is a whole team dedicated to manage puppet modules. They manage a bunch of modules for third party software. Why don't they maintain the PuppetDB module (or one for Puppetserver and Puppet Agent)? I asked that a few times during office hours but never got a proper answer.

Reply:
@GSPatton I have suggested we talk about this at the next VOX sync and it might be good for your product area to be aware and to help discuss

Reply:
added this PR as a talking point to the agenda (https://github.com/voxpupuli/community-triage/wiki)

Reply:
Puppetlabs's own puppetserver team having to work around the lack of maintenance of this module makes this preamble even more puzzling:

* https://github.com/puppetlabs/puppetserver/pull/2809
* https://github.com/puppetlabs/puppetserver/pull/2810

Reply:
This was discussed at length: https://github.com/voxpupuli/community-triage/wiki#february-13-2024

The short summary is:
* Historically there exists an internal version to support PuppetDB in PE
* Puppet uses this module for CI
* This preamble is intended to clarify the existing situation
* By the next community meeting (2024-02-13) we should have more clarity on the path forward